pkg/scheduler/objects/queue.go

```diff
@@ -957,12 +957,10 @@ func (sq *Queue) addChildQueue(child *Queue) error {
 // This can be executed multiple times and is only effective the first time.
 // This is a noop on an unmanaged queue.
 func (sq *Queue) MarkQueueForRemoval() {
-	// need to lock for write as we don't want to add a queue while marking for removal
```

Comment:
Note: this comment is not necessary. Config refresh is done under a partition lock, so the hierarchy cannot be modified while queues are being marked.